index.bs

```diff
@@ -64,9 +73,9 @@ url: http://aomediacodec.github.io/av1-spec/av1-spec.pdf#; spec: AV1; type: dfn;
 
 <h2 id="general">Scope</h2>
 
-[[!AV1]] defines the syntax and semantics of an [=AV1 bitstream=]. The <dfn>AV1 Image File Format</dfn> defined in this document supports the storage of a subset of the syntax and semantics of an [=AV1 bitstream=]. Specifically, only bitstreams with the <code>[=still_picture=]</code> flag set to 1 are supported. For some other configurations of AV1 bitstreams, the [[!AV1-ISOBMFF]] specification SHOULD be used.
-
-ISSUE: Is the restriction to have still_picture = 1 necessary?
+[[!AV1]] defines the syntax and semantics of an [=AV1 bitstream=]. The <dfn>AV1 Image File Format</dfn> (AVIF) defined in this document supports the storage of a subset of the syntax and semantics of an [=AV1 bitstream=] in a [[!HEIF]] file.
```

Comment:
`in a [[!MIAF]] file.`?

Reply:
This is a bit confusing, but I believe the AVIF spec is supposed define a basic mapping of AV1 to HEIF and then also define Profiles that require adherence to the MIAF spec. The MIAF spec in some sense is not a different file format but just a set of restrictions on HEIF.

It is possible, that someone may find a need to put AV1 in a HEIF container but use extra features that are not compatible with MIAF. In that case, they could still use the general provisions of the AVIF spec, but they would not implement any of the Profiles.

In other words, the idea is to make the profile-independent section of the AVIF spec reference HEIF instead of MIAF to allow for future flexibility.

Reply:
I support that the technology specification (how to do AV1 in this family) is AV1-in-HEIF, and that the profile restrictions are to match MIAF.